CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 # Unreleased
 
+## [7.0.1] - Tue Ago 15 2018
+### Change
+- Remove the Blackberry references since it is not so longer supported. 
```

Comment:
Change "since it is not so longer supported" to "since it is no longer supported"